SCClassLibrary/Common/Control/Server.sc

```diff
@@ -717,6 +718,14 @@ Server {
 		^Buffer.cachedBufferAt(this, bufnum)
 	}
 
+	defaultGroupIDForClientID { |id| ^nodeAllocator.numIDs * id + 1 }
+
+	sendDefaultGroupsFor { |ids|
+		ids.do { |id|
+			this.sendMsg("g_new", this.defaultGroupIDForClientID(id));
+		}
+	}
+
```

Comment:
nice.

Reply:
thanks :-)
We are preparing the workshop at AdK now, and this is really useful for making the setups robust for multiclients.